app/components/variable-height-textarea.js

```diff
@@ -5,18 +5,28 @@ export default Ember.TextArea.extend({
   attributeBindings: ["disabled"],
   disabled: false,
 
+   didDestroyElement: function () {
+     Ember.$('.liquid-container').css({
```

Comment:
is liquid container an ember thing ?

Reply:
`Ember.$('.liquid-container')` will fetch any liquid container across the whole app. Which sounds dangerous.
If this is really needed, make sure to only apply the changes on the liquid-container that matter (i'm guessing the one containing the textarea)

Reply:
`liquid-container` is just a class defined to the main `div`, after body on browse. the CSS which I'm applying at the time of destroyComponent is already defined. All I'm doing is unsetting at the time of component render, and resetting it back as soon as the component goes away.

Reply:
@patrixr i did not want to modify that thing globally so i took this approach of unsetting and resetting by using component lifecycle methods.